web-animations-1/Overview.bs

```diff
@@ -796,9 +831,22 @@ An animation's start time is initially <a>unresolved</a>.
 
 An <a>animation</a> also maintains a <dfn>hold time</dfn> <a>time value</a>
 which is used to fix the animation's output <a>time value</a>, called its
-<a>current time</a>, in circumstances such as pausing</a>.
+<a>current time</a>, in circumstances such as pausing.
 The <a>hold time</a> is initially <a>unresolved</a>.
 
+An <a>animation</a> also maintains a <dfn>hold phase</dfn>
+[=timeline phase|phase=] which is used by <a>animation effect</a> to calculate
+its correct phase in circumstances such as pausing. The <a>hold phase</a> is
+initially <a>unresolved</a>. Whenever <a>hold time</a> is changed,
+<a>hold phase</a> should be updated as follows:
```

Comment:
Nit: In general we should start a new line for each sentences. It makes subsequent diffs smaller.

Also, "should" has a special meaning in specs as something that can be ignored. Instead it would be better to simply make it descriptive:

"Whenever an animation's hold time is changed, its hold phase is updated as follows:"

Reply:
Deferring hold phase until a later PR.